.gitlab-ci.yml

```diff
@@ -0,0 +1,49 @@
+# paritytech/wasmi
+
+stages:
+  - benchmark
+
+default:
+  retry:
+    max: 2
+    when:
+      - runner_system_failure
+      - unknown_failure
+      - api_failure
+
+.rust-info-script:                 &rust-info-script
+  - rustup show
+  - cargo --version
+  - rustup +nightly show
+  - cargo +nightly --version
+  - cargo spellcheck --version
```

Comment:
not needed, right?

Reply:
Yes. It's just debug info. Will remove later

Reply:
please remove this line for merge